pkgs/development/python-modules/aqualogic/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyserial
+, pytestCheckHook
+, pythonOlder
```

Comment:
```suggestion
```

Reply:
Indeed, that's not needed, thanks. Removed.